source/guides/helpers/forms.md

```diff
@@ -201,6 +203,140 @@ Developers can customize attack handling.
 </form>
 ```
 
+### Pass A Value
```

Comment:
@davydovanton I think that there is still room for improvement in views. By looking at this updated guide one may think "OMG, should I use all this stuff to create a simple form?". Again, this looks overwhelming. The right thing to do is to try to improve the API, instead of offer this informations overload.